README.md

```diff
@@ -91,25 +93,45 @@ sudo mv engine_linux_amd64/srcd /usr/local/bin/
 
 **Windows:**
 
-_Windows support is coming soon!_
+Extract the tar file with the tool you prefer.
+
+Open your powershell and move the `srcd.exe` file it into any path listed in `PATH`.
```

Comment:
Can we simplify this section? Is there a default path we can use here to make it ready to copy&paste?

Reply:
Here's the content of `PATH` on the dev machine:
```powershell
PS C:\Users\Windows10\go> $env:path.split(";")
C:\ProgramData\DockerDesktop\version-bin
C:\Program Files\Docker\Docker\Resources\bin
C:\Windows\system32
C:\Windows
C:\Windows\System32\Wbem
C:\Windows\System32\WindowsPowerShell\v1.0\
C:\Windows\System32\OpenSSH\
C:\Program Files\Git\cmd
C:\Go\bin
C:\ProgramData\chocolatey\bin
C:\Users\Windows10\AppData\Local\Microsoft\WindowsApps
C:\Users\Windows10\go\bin
```

I think that `C:\Windows\system32` could be a nice candidate.

Reply:
When I was young, most of apps suggested you add your directory into your path env... It could be an alternative...
```powershell
SET PATH=%PATH%;{DIR_WHERE_SRCD.EXE_WAS_UNPACKED};
```

Reply:
@dpordomingo yup that is an alternative. I avoided explaining that just to make the quickstart easier.

Reply:
manually copying a downloaded thing into `system32`? wow... it might be scary...

Reply:
> it might be scary...
😂 
I actually don't know windows, but I guess it doesn't differ too much from copying a downloaded thing into `/usr/local/bin` in unix systems. But maybe I'm wrong.

Reply:
[Go installation guide](https://golang.org/doc/install/source#install) in Windows suggests you to add go bin directory in your `PATH`, so it seems a default behavior for windows developers...

Reply:
I think that `C:\Windows\system32` is not a user directory. I wonder if even the antivirus could complain about it...

Reply:
Ok. We just have to decide where to put it then.

Reply:
> I wonder if even the antivirus could complain about it...

Good point! 👍 
I haven't thought about that, it's been years that I don't run an antivirus...

Reply:
What about `C:\Program Files\Engine` (maybe too generic)? or `C:\Program Files\srcd`?

Reply:
If there is no permissions problem, both lgtm

Reply:
I prefer `C:\Program Files\srcd`, it sounds more generic, and it could be used for other tools we may release in the future.